docs/swagger.yaml

```diff
@@ -82,6 +82,96 @@ paths:
       tags:
         - Merchant redirect QR
     parameters: []
+    get:
+      summary: Get all QRs
+      operationId: GetAllMerchantRedirectQrs
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerchantRedirectQrResponseGetAll'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+      description: Get all QRs created for one saleunit
+      tags:
+        - Merchant redirect QR
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+  '/v1/merchantRedirect/{id}':
+    put:
```

Comment:
Should we consider just having one get-endpoint, and accept Id or a list of id as a query parameter? Thats a pretty RESTful and normal way to handle get endpoints